README.md

```diff
@@ -35,28 +35,114 @@ eliminates the need to manually choose identifiers for your nodes/relationships/
 
 ### Referencing The Path
 
-If you'd like to reference the path from your `MATCH` expression in your `WHERE` expression, you can do
+If you'd like to reference the path from your match expression in your where expression, pass a
+`Path => Where` to the `where` method
 
 ```scala
 val cypher = startNode --> AnyNode() where { path =>
-		...
-	} returns startNode
-
+    ...
+  } returns startNode
 ```
 
 ### Getting Return Columns
 
 When building out your persistence layer, it is likely you will need to know the identifier used in your
-RETURN expressions. This can be obtained with:
+return expressions. This can be obtained with:
 
 ```scala
-query.getReturnColumns
+cypher.getReturnColumns
```

Comment:
i think both are valid, but i guess i don't feel super strongly about it. might change later.